blob/blob.go

```diff
@@ -1089,14 +1090,18 @@ func OpenBucket(ctx context.Context, urlstr string) (*Bucket, error) {
 	return defaultURLMux.OpenBucket(ctx, urlstr)
 }
 
-func wrapError(b driver.Bucket, err error) error {
+func wrapError(b driver.Bucket, err error, key string) error {
 	if err == nil {
 		return nil
 	}
 	if gcerr.DoNotWrap(err) {
 		return err
 	}
-	return gcerr.New(b.ErrorCode(err), err, 2, "blob")
+	msg := "blob"
+	if key != "" {
+		msg += fmt.Sprintf(" (key %s)", key)
```

Comment:
nit: use `%q` to quote `key` since it might be something like key1 -> key2 in the copy scenario.

Reply:
Done.